hyundai_kia_connect_api/VehicleManager.py

```diff
@@ -69,13 +69,12 @@ def check_and_force_update_vehicles(self, force_refresh_interval: int) -> None:
         for vehicle_id in self.vehicles.keys():
             vehicle: Vehicle = self.get_vehicle(vehicle_id)
             _LOGGER.debug(
-                f"time diff - {(started_at_utc - vehicle.last_updated_at).total_seconds()}"
+                f"{DOMAIN} - time diff - {(started_at_utc - vehicle.last_updated_at).total_seconds()}"
             )
             if (
                 started_at_utc - vehicle.last_updated_at
             ).total_seconds() > force_refresh_interval:
                 self.force_refresh_vehicle_state(vehicle)
-                self.update_vehicle_with_cached_state(vehicle)
```

Comment:
By removing this line, we acknowledge the fact that all the necessary steps were already handled in force_refresh_vehicle_state, and this call was redundant, right?

Reply:
the necessary steps being:
- pulling fresh vehicle data from the API (not just triggering a refresh from the server to the vehicle)
- filling the Vehicle object with the API response

sorry if my comment in confusing. I'm just wondering why this line was there to begin with.

Reply:
Yes*.  Both canada and EU were adjusted to actually take the force update response and process.  The old approach was ignore the response and call another cached state call as above.   

Now USA has not been adjusted for this as they aren't well supported since we don't have testers for it.  I just dropped a PR for Kia USA that would account for this by shifting the above logic into the region.  I will do the same for Hyundai USA which doesn't even have force update implemented - no one has sniffed this traffic before including bluelinky.